Cabal-syntax/src/Distribution/Compat/CharParsing.hs

```diff
@@ -355,3 +356,7 @@ munch = many . satisfy
 skipSpaces1 :: CharParsing m => m ()
 skipSpaces1 = skipSome space
 {-# INLINE skipSpaces1 #-}
+
+skipSpaces :: CharParsing m => m ()
```

Comment:
Remove. There is `spaces :: CharParsing m => m ()`.

Reply:
@gbaz: what about that one?

Reply:
oh, I'm sorry, silly github has shown me outdated patch